test/e2e/ingress/vanilla_ingress_test.go

```diff
@@ -162,12 +172,16 @@ var _ = Describe("vanilla ingress tests", func() {
 					Controller: "kubernetes.io/nginx",
 				},
 			}
+			annotation := map[string]string{
+				"alb.ingress.kubernetes.io/scheme": "internet-facing",
+			}
+			if tf.Options.IPFamily == "IPv6" {
+				annotation["alb.ingress.kubernetes.io/ip-address-type"] = "dualstack"
```

Comment:
don't we need ip target type as well?

Reply:
For this test case no ALB will be created, so I didn't add target ip annotation here, it won't be used.